### PR TITLE
Adapt OOMKilling pattern to old and new Linux kernels

### PR DESCRIPTION
**Pull request by tosi3k**
Context: https://github.com/kubernetes/node-problem-detector/issues/480.

Tl;dr: kernel commit https://github.com/torvalds/linux/commit/bbbe48029720d2c6b6733f78d02571a281511adb removed logging the `Kill process %d (%s) score %u or sacrifice child\n` OOM message part we look for when tracing the kernel log in the kernel monitor. Because of that, the change removes that part from the `kernel-monitor(-filelog).json` regex pattern to handle both old and new kernel versions.

Successfully tested the change using https://github.com/kubernetes/node-problem-detector#try-it-out instruction (appropriate K8s events are emitted for both old and new OOM message formats).

Fixes https://github.com/kubernetes/node-problem-detector/issues/480.

### CODE REVIEW

**Comment by tosi3k** [comment]
Friendly ping for a review request @andyxning @xueweiz :).

When could one expect a new release of NPD (incorporating this fix)?

**Comment by Random-Liu** [comment]
/lgtm
/approve

Thanks for the fix!
